### ISSUE TITLE
Document that vector-web is a static webapp (and /packages exists)

### ISSUE THREAD

**Issue by killua-eu** [vector-im/vector-web#1693]
Hi all, this is most probably a quick documentation request ticket: how can one run vector over https? how do I pass encryption keys etc? Thanks in advance.

**Comment by ara4n** [comment]
Vector is an entirely static app, and runs just fine hosted from HTTPS.

To access your homeserver via HTTPS, it just needs to have a valid TLS 
certificate (or your browser needs to have been told to trust an invalid 
one).

**Comment by killua-eu** [comment]
sorry, put the question wrong: can I run vector-web without ssl termination and proxy_passing unencrypted traffic to vector? if so, how?

**Comment by ara4n** [comment]
Yes.  Vector is just a static bunch of html, css & js.  You can serve it 
however you like (e.g. on a plain HTTP host) and then proxypass that 
through an SSL LB.

On 26/06/2016 23:48, killua-eu wrote:
> sorry, put the question wrong: can I run vector-web without ssl
> termination and proxy_passing unencrypted traffic to vector? if so,
> how?
>
> — You are receiving this because you commented. Reply to this email
> directly, view it on GitHub
> <https://github.com/vector-im/vector-web/issues/1693#issuecomment-228618866>,
>
>
or mute the thread
> <https://github.com/notifications/unsubscribe/ABO_vcT_vmyydQzEPIGrXfxjUfCvFiK-ks5qPtd3gaJpZM4I-Lw7>.
>

**Comment by killua-eu** [comment]
OK, so there's an blatant gap in my expectations in how npm built things work, because you kept loosing me on something obviously clear. I'll document my firsthand exploration of my stupidity to help all other poor lost souls in my shoes :) So, the docs states

```
git clone https://github.com/vector-im/vector-web.git
cd /opt/vector-web
npm install
nano /opt/vector-web/config.json
npm run build && npm start
```

Your reply brought me to google "what does npm start do" and I figured out that it does exactly what it finds on the line  **cat /opt/vector-web/package.json  | grep \"start\":**

`"start": "parallelshell \"npm run start:js\" \"npm run start:skins:css\" \"http-server -c 1 vector\"",`

So basically it seems it just runs a http-server for the content of /opt/vector-web/vector, so serving the content of /opt/vector-web/vector with apache or nginx will yield just about the same result. Yes, it is as simple as pointing the document root to /opt/vector-web/vector and restarting your webserver, your decision if you keep it on the default 8080 port, 80 port or put it on 443. 

Any chance documenting this in the readme, or is this a super common knowledge?

**Comment by ara4n** [comment]
Yup, this should be much clearer. It's a fairly common idiom in React but we can spell it out much better.

**Comment by ara4n** [comment]
these days the README does a better job (i hope!) of spelling this out.
